backend/src/routes/sessions.ts

```diff
@@ -33,26 +39,36 @@ const router = express.Router();
 
 router.post(
   "/sessions",
-  validateReqBodyWithCake(CreateSessionRequestBodyCake),
+  [validateReqBodyWithCake(CreateSessionRequestBodyCake), verifyAuthToken],
```

Comment:
We can just use `req.body.uid` to get the menteeID cauz its a mentee that would always access this page. Then use the pairing schema to get the `mentorID`